Include/internal/pycore_gc.h

```diff
@@ -109,11 +109,15 @@ static inline void _PyObject_GC_SET_SHARED_INLINE(PyObject *op) {
 
 /* Bit flags for _gc_prev */
 /* Bit 0 is set when tp_finalize is called */
-#define _PyGC_PREV_MASK_FINALIZED  (1)
+#define _PyGC_PREV_MASK_FINALIZED  1
 /* Bit 1 is set when the object is in generation which is GCed currently. */
-#define _PyGC_PREV_MASK_COLLECTING (2)
+#define _PyGC_PREV_MASK_COLLECTING 2
+
+/* Bit 0 is set if the object belongs to old space 1 */
+#define _PyGC_NEXT_MASK_OLD_SPACE_1    1
+
 /* The (N-2) most significant bits contain the real address. */
```

Comment:
This comment confused me. It seems there are different places where these two flags (`_PyGC_PREV_SHIFT` and `_PyGC_PREV_MASK`) are used -- sometimes there is a counter of refs, which is indeed shifted left 2 bits; sometimes there is an address, which is not shifted but has the lower two bits cleared. Maybe you can explain this a bit better?

(It's also confusing that there are both `PREV` and `NEXT` mask definitions, but sometimes the `PREV` mask is used with a member named `_gc_next`. This may all be for historical reasons, but doesn't help understanding the code.)

Reply:
I think that comment belonged somewhere in an earlier version. I've deleted it.